lib/internal/Magento/Framework/Filesystem/Driver/File.php

```diff
@@ -660,6 +660,9 @@ public function fileUnlock($resource)
      */
     public function getAbsolutePath($basePath, $path, $scheme = null)
     {
+        if (strlen($path) > 0 && $this->fixSeparator($path)[0] === '/') {
```

Comment:
The behaviour of `getAbsolutePath()` surprised me and I actually think the change you've made here should be part of Magento 2 regardless of this particular PR. Is there any reason not to submit this as its own PR?